prover/zkevm/prover/publicInput/public_input.go

```diff
@@ -302,4 +302,26 @@ func (pi *PublicInput) generateExtractor(comp *wizard.CompiledIOP) {
 		L2MessageServiceAddrHi:   accessors.NewFromPublicColumn(pi.Aux.logSelectors.L2BridgeAddressColHI, 0),
 		L2MessageServiceAddrLo:   accessors.NewFromPublicColumn(pi.Aux.logSelectors.L2BridgeAddressColLo, 0),
 	}
+
```

Comment:
@arijitdutta67 @Soleimani193 

This is how we concretely register public inputs

Reply:
TODO, we should also modify the circuit to account for that